daal4py/sklearn/linear_model/_linear.py

```diff
@@ -157,25 +158,27 @@ def _fit_linear(self, X, y, sample_weight=None):
     self.fit_shape_good_for_daal_ = \
         bool(X.shape[0] > X.shape[1] + int(self.fit_intercept))
 
-    daal_ready = self.fit_shape_good_for_daal_ and not sp.issparse(X) and \
-        sample_weight is None
+    _patching_status = PatchingConditionsChain(
+        "sklearn.linear_model.LinearRegression.fit")
+    _patching_status.and_conditions([
+        (not sp.issparse(X), "input is sparse"),
+        (self.fit_shape_good_for_daal_, "shape is not good for onedal"),
+        (sample_weight is None, "sample_weight is not None")])
+
     if sklearn_check_version('0.22') and not sklearn_check_version('0.23'):
-        daal_ready = daal_ready and dtype in [np.float32, np.float64]
+        _patching_status.and_conditions([
+            (dtype in [np.float32, np.float64],
+                f"dtype is '{dtype}' while float32/float64 are supported")])
```

Comment:
Please align message for floating point types, below you have something like **floating point type is incorrect**